app/src/pages/inside/logsPage/logsGrid/logsGrid.jsx

```diff
@@ -0,0 +1,228 @@
+            component: AttachmentColumn,
+            customProps: {
+              consoleView: true,
+            },
+          },
+          {
+            id: timeColumnId,
+            sortable: true,
+            customProps: {
+              consoleView: true,
+            },
+            component: MessageColumn,
+          },
+          {
+            id: 'mobileTime',
+            component: TimeColumn,
+            customProps: {
+              mobile: true,
+            },
+          },
+          {
+            id: 'mobileAttachment',
+            component: AttachmentColumn,
+            customProps: {
+              mobile: true,
+            },
+          },
+        ]
+      : [
+          {
+            id: 'logMessage',
+            title: {
+              component: LogMessageSearch,
+              componentProps: {
+                filter: this.props.filter,
+                onFilterChange: this.props.onFilterChange,
+              },
+            },
+            maxHeight: 200,
+            component: MessageColumn,
+          },
+          {
+            id: 'attachment',
+            component: AttachmentColumn,
+          },
+          {
+            id: timeColumnId,
+            title: {
+              full: this.props.intl.formatMessage(messages.timeColumnTitle),
+            },
+            sortable: true,
+            component: TimeColumn,
+          },
+          {
+            id: 'mobileAttachment',
+            title: {
+              component: () => <div className={cx('no-header')} />,
+            },
+            component: AttachmentColumn,
+            customProps: {
+              mobile: true,
+            },
+          },
+        ];
+
+  getLogRowClasses = (value) => {
+    const { consoleView } = this.props;
+
+    return {
+      log: true,
+      'error-row': !consoleView && (value.level === ERROR || value.level === FATAL),
+      [cx('row-console')]: consoleView,
+      [cx('error-row-console')]: consoleView && (value.level === ERROR || value.level === FATAL),
+    };
+  };
+
+  render() {
+    const {
+      intl,
+      logItems,
+      loading,
+      sortingColumn,
+      sortingDirection,
+      onChangeSorting,
+      consoleView,
+      filter,
+      onFilterChange,
+    } = this.props;
+
+    return (
+      <div className={cx('logs-grid')}>
+        {consoleView && (
+          <div className={cx('console-view-header')}>
+            <div
+              className={cx('time-header', {
+                [`sorting-${sortingDirection.toLowerCase()}`]: sortingDirection,
+                'sorting-active': sortingColumn === timeColumnId,
+              })}
+              onClick={() => onChangeSorting(timeColumnId)}
+            >
+              {intl.formatMessage(messages.timeColumnTitle)}
+              <div className={cx('arrow')}>{Parser(ArrowIcon)}</div>
+            </div>
+            <LogMessageSearch filter={filter} onFilterChange={onFilterChange} />
+          </div>
```

Comment:
I think it's better to move it in separate function

Reply:
Done